Snowflake.Data.Tests/SFPutGetTest.cs

```diff
@@ -107,14 +112,250 @@ public void TearDown()
                 }
             }
             
-            // Delete temp files
-            File.Delete(_inputFilePath);
-            File.Delete(_outputFilePath);
+            // Delete temp files if necessary
+            if (_filesToDelete == null)
+                return;
+            
+            foreach (var file in _filesToDelete)
+            {
+                File.Delete(file);
+            }
+        }
+
+        [Test]
+        public void TestPutFileAsteriskWildcard()
+        {
+            // Prepare the data files to be copied
+            var absolutePath = $"{Path.GetTempPath()}{Guid.NewGuid()}";
+            var files = new List<string> {
+                $"{absolutePath}_one.csv",
+                $"{absolutePath}_two.csv",
+                $"{absolutePath}_three.csv"
+            };
+            PrepareFileData(files);
+            
+            // Set the PUT query variables
+            _inputFilePath = $"{absolutePath}*";
+            _internalStagePath = $"@{_schemaName}.{_stageName}";
+
+            using (var conn = new SnowflakeDbConnection(ConnectionString))
+            {
+                conn.Open();
+                PutFile(conn);
+                // Verify that all files have been uploaded
+                VerifyFilesAreUploaded(conn, files, _internalStagePath);
+            }
+        }
+        
+        [Test]
+        public void TestPutFileAsteriskWildcardWithExtension()
+        {
+            // Prepare the data files to be copied
+            var absolutePath = $"{Path.GetTempPath()}{Guid.NewGuid()}";
+            var files = new List<string> {
+                $"{absolutePath}_one.csv",
+                $"{absolutePath}_two.csv",
+                $"{absolutePath}_three.csv"
+            };
+            PrepareFileData(files);
+            // Create file with the same name structure but with a different extension
+            PrepareFileData($"{absolutePath}_four.txt");
+
+            // Set the PUT query variables
+            _inputFilePath = $"{absolutePath}*.csv";
+            _internalStagePath = $"@{_schemaName}.{_stageName}";
+
+            using (var conn = new SnowflakeDbConnection(ConnectionString))
+            {
+                conn.Open();
+                PutFile(conn);
+                // Verify that all files have been uploaded
```

Comment:
all csv files? do we need such comments at all?

Reply:
Removed the unnecessary comments.